src/pykeen/cli.py

```diff
@@ -234,6 +235,25 @@ def _help_optimizers(tablefmt: str, link_fmt: Optional[str] = None):
     )
 
 
+@ls.command()
+@tablefmt_option
+def lr_schedulers(tablefmt: str):
+    """List optimizers."""
+    click.echo(_help_lr_schedulers(tablefmt))
+
+
+def _help_lr_schedulers(tablefmt: str, link_fmt: Optional[str] = None):
+    lines = _get_lines_alternative(
```

Comment:
@lvermue I'm so sorry you had to figure this out. This code is way too abstract for its own good.